projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -1,12 +1,37 @@
 import { Component, Element, Event, EventEmitter, Listen, Prop, State } from '@stencil/core';
-import { DotCMSContentTypeField, DotFieldStatus } from '../../models';
+import { DotCMSContentTypeField, DotFieldStatus, DotCMSKeyValueField } from '../../models';
 import { DotFormFields } from './dot-form-fields';
 import { getClassNames, getOriginalStatus } from '../../utils';
 
+const fieldParamsConversionToBE = {
+    'Key-Value': (values: string): { [key: string]: string } => {
+        const returnValue = {};
+        values.split(',').forEach((item: string) => {
+            const [key, value] = item.split('|');
+            returnValue[key] = value;
+        });
+        return returnValue;
+    }
+};
+
+const fieldParamsConversionFromBE = {
+    'Key-Value': (field: DotCMSKeyValueField) => {
+        if (field.defaultValue && typeof field.defaultValue !== 'string') {
+            field.defaultValue = Object.keys(field.defaultValue)
+                .map((e) => {
```

Comment:
This code and [this](https://github.com/dotCMS/core-web/pull/1008/files#diff-3923e7309ccdea946eb1f2dc46e564e7R188) is the same, can we move this to their own function helper?